tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,69 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd alertmanager_ip grafana_ip prometheus_ip
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=edge --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	echo "Wait for all coslite bundle charms to be active"
+	wait_for "active" '.applications["alertmanager"] | ."application-status".current' 1800
+	wait_for "active" '.applications["grafana"] | ."application-status".current' 1800
+	wait_for "active" '.applications["loki"] | ."application-status".current' 1800
+	wait_for "active" '.applications["prometheus"] | ."application-status".current' 1800
+	wait_for "active" '.applications["traefik"] | ."application-status".current' 1800
+
+	echo "Check that all relations specified in the bundle exist"
+	wait_for 5 '.applications["alertmanager"] | ."relations" | length'
+	wait_for 3 '.applications["grafana"] | ."relations" | length'
+	wait_for 5 '.applications["loki"] | ."relations" | length'
+	wait_for 6 '.applications["prometheus"] | ."relations" | length'
+	wait_for 2 '.applications["traefik"] | ."relations" | length'
```

Comment:
I'm unsure about this check as well. It seems overly specific and again a little fragile. It is very possible the number of relations will change

Reply:
What did you have in mind? My thinking is that, if things change in future we will refactor the tests, because we will also need to update the overlay files bundled in this test.

Reply:
I'm a little unsure really if this check adds anything really. For sure get a second opinion

Either way, these should be changed to a `check` since the bundle has already been deployed

Reply:
Oh here's an idea. I think it would be better to check the expected offers exist. These relations are purely internal workings, but offers are part of the external interface. 

I would replace these with checks that we have the expected offers

Reply:
Sure thing, a general check for the length of the offers' endpoints as dictated by the offers overlay file would suffice.